### DISCUSSION TITLE
Dados para a API

### DISCUSSION THREAD

**Discussion by ttsoares** [raulfdm/taco-api#37] [Q&A] [answered]
Gostaria de subir essa API na minha conta do Heroku. 
Mas, do que entendi, aqui só tem o front...  Isso ?

Onde posso achar o que precisa para instalar lá o back-end ?
Sei como fazer. Só preciso dos fontes e SQLs...

**Comment by raulfdm** [comment, marked as answer]
Fala @ttsoares.

Da uma olhada na versão 2 que eu acabei de lançar, talvez vai te ajudar.

O projeto é interamente backend, não tem nada de front.

https://taco-api.netlify.app/#usando-em-producao
